app/models/tagging.rb

```diff
@@ -42,4 +43,10 @@ def update_search
     # in multiple fandoms.
     tagger.enqueue_to_index if tagger.taggings_count < ArchiveConfig.TAGGINGS_COUNT_REINDEX_LIMIT
   end
+
+  def collection_tag_limit
+    return unless taggable.class == Collection && taggable.tags.count >= ArchiveConfig.COLLECTION_TAGS_MAX
+
+    errors.add(:tags, ts("Sorry, you can only save %{maximum} collections tags.", maximum: ArchiveConfig.COLLECTION_TAGS_MAX))
```

Comment:
We had a quick discussion about this error message and decided it could maybe be worded a bit differently. Could you change it to: `Sorry, a collection can only have %{maximum} tags.`? Thank you!

Reply:
Sure thing!